nupm/install.nu

```diff
@@ -120,15 +103,15 @@ def install-path [
                 log debug $"installing scripts for package ($package.name)"
 
                 $package.scripts
-                | install-scripts $pkg_dir (script-dir --ensure) --force  $force
+                | install-scripts $pkg_dir (script-dir --ensure) --force $force
             }
         },
         "script" => {
             log debug $"installing scripts for package ($package.name)"
 
-            $package.scripts?
-            | default [ ($pkg_dir | path join $"($package.name).nu") ]
-            | install-scripts $pkg_dir (script-dir --ensure) --force  $force
+            [ ($pkg_dir | path join $"($package.name).nu") ]
+            | append ($package.scripts? | default [])
```

Comment:
same, curious to see what the bug was here :yum:

Reply:
Here I think the problem was that the list of scripts to install was wrong in some cases in the old code.